x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/connectors_dropdown/index.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiSuperSelect, EuiIcon, EuiSuperSelectOption } from '@elastic/eui';
+import styled from 'styled-components';
+
+import * as i18n from '../translations';
+
+const ICON_SIZE = 'm';
+
+const EuiIconExtended = styled(EuiIcon)`
+  margin-right: 13px;
+`;
+
+const ConnectorsDropdownComponent: React.FC = () => {
+  const connectors: Array<EuiSuperSelectOption<string>> = [
```

Comment:
I think this const `connectors` can live outside of this component, just above to avoid re-render